<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CodeBox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>740</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.25, y1:0.563, x2:1, y2:0, stop:0 rgba(55, 185, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <widget class="QTextEdit" name="input_textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>850</width>
       <height>180</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="output_textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>850</width>
       <height>190</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="encode_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>190</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);
background-color: rgb(170, 255, 127);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Шифровать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="decode_button">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>190</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);
background-color: rgb(255, 255, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Расшифровать</string>
     </property>
    </widget>
    <widget class="QLabel" name="keylabel">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>250</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(0, 0, 0,0);</string>
     </property>
     <property name="text">
      <string>Ключ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Keyfild">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>240</y>
       <width>361</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255,255,255);</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="randomkey_checkBox">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>300</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>171</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font: 100 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(0,0,0,0);</string>
     </property>
     <property name="text">
      <string>Случайный ключ</string>
     </property>
    </widget>
    <widget class="QLabel" name="input_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(0,0,0,0);</string>
     </property>
     <property name="text">
      <string>Входные данные</string>
     </property>
    </widget>
    <widget class="QLabel" name="ouput_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(0,0,0,0);</string>
     </property>
     <property name="text">
      <string>Выходные данные</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_button_input">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);
background-color: rgb(255, 85, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_button_output">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>550</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);
background-color: rgb(255, 85, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="show_key_checkBox">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>300</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 100 12pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;
font: 14pt &quot;MS Shell Dlg 2&quot;; color: green;
background-color: rgba(0,0,0,0);</string>
     </property>
     <property name="text">
      <string>Показать ключ</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
